resources/migrations/000_migrations.yaml

```diff
@@ -8339,6 +8339,23 @@ databaseChangeLog:
               WHERE base_type IN ('ArrayField', 'BigIntegerField', 'BooleanField', 'CharField', 'DateField',
                                   'DateTimeField', 'DecimalField', 'DictionaryField', 'FloatField', 'IntegerField',
                                   'TextField', 'TimeField', 'UUIDField', 'UnknownField');
+  - changeSet:
+      id: 306
```

Comment:
Seems like 306 was already claimed, but never used (?) - see #migrations channel - just want to avoid conflict